MANIFEST.in

```diff
@@ -2,7 +2,12 @@ include README.md
 include LICENCE
 include scopesim/defaults.yaml
 include scopesim/vega.fits
-prune scopesim/tests*
+include scopesim/tests/mocks/*/*.yaml
```

Comment:
I would like to restrict this to mocks/basic_instrument/*.*  ... unless you need all the other data files for whatever tests. Do you?
I think all other folders (should) only be referenced by the test suite.

An alternative may be:
prune scopesim/tests*
graft scopesim/tests*/basic_instrument

I'll try it out locally and let you know

Reply:
Oops, I meant

prune scopesim/tests*
graft scopesim/tests/mocks/basic_instrument

Reply:
I changed your suggestion to this:

prune scopesim/tests*
graft scopesim/tests/mocks/basic_instrument*
include scopesim/tests/mocks/load_basic_instrument.py

Do you need any of the files from the other mock directories?

Reply:
Thanks, this is better.